modules/local/krona.nf

```diff
@@ -8,15 +8,16 @@ process KRONA {
 
     input:
     tuple val(meta), path(report)
-    path  "taxonomy/taxonomy.tab"
+    path(taxonomy_file)
 
     output:
-    path "*.html"       , emit: html
-    path "versions.yml" , emit: versions
+    tuple val(meta), path("*.html") , emit: html
+    path "versions.yml"             , emit: versions
 
     script:
+    taxonomy_folder = taxonomy_file.getParent()
     """
-    ktImportTaxonomy "$report" -tax taxonomy
+    ktImportTaxonomy "$report" -tax ${taxonomy_folder}
```

Comment:
```suggestion
    ktImportTaxonomy "$report" -tax .
```

If the `.tab` file is provided, it is staged directly in the working directory. We can just indicate `./` as the directory for `-tax`, and gets around all of these problems! (tested and works)!